.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
It would be great to use the same Prettier config than Gatsby (https://github.com/gatsbyjs/gatsby/blob/master/.prettierrc) and clean (remove parameters) the `format` command in the package.json.